x-pack/plugins/cases/public/components/case_view/components/case_view_alerts.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { Case } from '../../../../common';
+import { useKibana } from '../../../common/lib/kibana';
+import { getManualAlertIds, getFeatureIdsFromAlertIndices } from '../../user_actions/helpers';
+
+interface CaseViewAlertsProps {
+  caseData: Case;
+}
+export const CaseViewAlerts = ({ caseData }: CaseViewAlertsProps) => {
+  const { triggersActionsUi } = useKibana().services;
+
+  const alertIdsQuery = useMemo(
+    () => ({
+      ids: {
+        values: getManualAlertIds(caseData.comments),
+      },
+    }),
+    [caseData.comments]
+  );
+  const alertFeatureId = useMemo(
+    () => getFeatureIdsFromAlertIndices(caseData.comments),
+    [caseData.comments]
+  );
+
+  const alertStateProps = useMemo(() => {
+    const configurationId = alertFeatureId.includes('siem') ? 'securitySolution' : 'observability';
```

Comment:
Consumers of Cases pass an `appId` variable which denotes the ID of the application that uses cases. For cases on the stack management is `cases`. Do you think is better to use `appId` or we should rely on the `alertFeatureId`?

Reply:
Resolve by using cases owner